index.ts

```diff
@@ -15,49 +13,39 @@ async function run() {
     const client = new AirtopClient({
       apiKey: AIRTOP_API_KEY,
     });
+
     const createSessionResponse = await client.sessions.create({
       configuration: {
         timeoutMinutes: 5,
       },
     });
 
-    const session = createSessionResponse.data;
-    console.log('Created airtop session', session.id);
+    const sessionId = createSessionResponse.data.id;
+    console.log('Created airtop session', sessionId);
 
     if (!createSessionResponse.data.cdpWsUrl) {
       throw new Error('Unable to get cdp url');
     }
 
-    // Connect to the browser
-    const cdpUrl = createSessionResponse.data.cdpWsUrl;
-    const browser: Browser = await puppeteer.connect({
-      browserWSEndpoint: cdpUrl,
-      headers: {
-        Authorization: `Bearer ${AIRTOP_API_KEY}` || '',
-      },
-    });
-    console.log('Connected to browser');
+    // Create a new window and navigate to the URL
+    const windowResponse = await client.windows.create(
+      sessionId,
+      { url: TARGET_URL }
+    );
 
-    // Open a new page
-    const page: Page = await browser.newPage();
-    
-    // Navigate to the URL
-    console.log('Navigating to URL', TARGET_URL);
-    await page.goto(TARGET_URL);
-    const windowInfo = await client.windows.getWindowInfoForPuppeteerPage(session, page, {
+    const windowInfo = await client.windows.getWindowInfo(sessionId, windowResponse.data.windowId, {
       disableResize: true, // Prevents the browser window from being resized when loading a live view, which might impact the agent's ability to scrape or summarize content
-    }); 
+    });
     
     console.log('See a live view of the browser window at', chalk.blueBright(windowInfo.data.liveViewUrl));
     const windowId = windowInfo.data.windowId;
 
     console.log('Summarizing content...');
-    const contentSummary = await client.windows.promptContent(session.id, windowId, { prompt: 'Summarize the content of the page in 1 paragraph' }); // Note that scrapeContent is also available to do a clean scrape of the page content
+    const contentSummary = await client.windows.promptContent(sessionId, windowId, { prompt: 'Summarize the content of the page in 1 paragraph' }); // Note that scrapeContent is also available to do a clean scrape of the page content
     console.log('Content summary:\n\n', chalk.green(contentSummary.data.modelResponse));
 
     // Clean up
-    await browser.close();
-    await client.sessions.terminate(session.id);
```

Comment:
@dreamerns should we call `client.windows.close` for cleanup here?

Reply:
@pciporin25 I guess we can just to show more of the API, but it doesn't really matter right? You don't have to close the window before closing the browser.